integration_tests/__tests__/coverage_report-test.js

```diff
@@ -18,7 +18,11 @@ it('outputs coverage report', () => {
   const {stdout, status} = runJest('coverage_report', ['--coverage']);
   const coverageDir = path.resolve(__dirname, '../coverage_report/coverage');
 
-  // should be no `setup.file` in the coverage report. It's ignored
+  // - should be no `setup.file` in the coverage report. It's ignored
+  // - sum_dependency.js mocked and the real module is never required, but
+  //  the report should still have its (0% lines) coverage
+  // - not-required-in-test-suite.js never required or mocked. Report should
+  //  still have its (0%) coverage
```

Comment:
would you mind rewriting this a bit? It's hard for me to read and understand. How about:

```js
// the setup file is ignore and should not be in the coverage report.
// sum_dependency.js is mocked and the real module is never required but is listed with 0 % coverage.
// not-required-in-test-suite.js is not required but it is listed with 0 % coverage.
```